src/commonMain/kotlin/exchange.dydx.abacus/calculator/V2/TradeInput/TradeInputMarketOrderCalculator.kt

```diff
@@ -178,15 +187,19 @@ internal class TradeInputMarketOrderCalculator() {
 
     private fun createMarketOrderFromBalancePercent(
         balancePercent: Double,
+        marginMode: MarginMode,
         freeCollateral: Double,
-        maxMarketLeverage: Double,
-        subaccount: InternalSubaccountState?,
+        tradeLeverage: Double,
         orderbook: List<InternalOrderbookTick>?,
         stepSize: Double,
     ): TradeInputMarketOrder? {
-        val cappedPercent = min(balancePercent, MAX_FREE_COLLATERAL_BUFFER_PERCENT * balancePercent)
```

Comment:
oops my bad, should've been `min(balancePercent, MAX_FREE_COLLATERAL_BUFFER_PERCENT)`